examples/polymer-cli/src/stories/addon-backgrounds.stories.js

```diff
@@ -0,0 +1,11 @@
+import { storiesOf } from '@storybook/polymer';
+import backgrounds from '@storybook/addon-backgrounds';
+
+storiesOf('Addon|Backgrounds', module)
+  .addDecorator(
+    backgrounds([
```

Comment:
This should be `withBackgrounds` as per https://github.com/storybooks/storybook/commit/460d5acf6a10450a1c0d46e53ccaef09a46d31fc right?